framework/src/iotbus/iotbus_spi.c

```diff
@@ -75,44 +74,50 @@ iotbus_spi_context_h iotbus_spi_open(unsigned int bus, const struct iotbus_spi_c
 		return NULL;
 	}
 
+	/*	Open driver */
+	char spi_dev[16] = { 0, };
+	snprintf(spi_dev, 16, "/dev/spi-%d", bus);
```

Comment:
May be sizeof(spi_dev)  would be better than hard coding 2nd parameter

Reply:
those two variables are close, so there would be no mistakes.